### DISCUSSION TITLE
Explore/Loki: Remove double quotes and the escapes for detected string fields

### DISCUSSION THREAD

**Discussion by honganan** [grafana/grafana#48893] [Dashboards, Panels & Visualizations]
The Explore logs panel shows `log labels` and `Detected fields` When click to watch details of a log line.

Now the `log labels` part shows string value without double quotes and is very clear, While `Detected fields` shows string with double quotes, which may escape fields' content. Hope `Detected fields` do the same like `log labels` when string value detected. Like Kibana did.

`log labels`:
![image](https://user-images.githubusercontent.com/4892550/167797429-794e822b-a1e9-4921-a924-ecf033eebfb8.png)

`Detected fields`:
![image](https://user-images.githubusercontent.com/4892550/167797737-c23aefb0-9a0d-4ead-a4a9-cce4771ff69a.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
